examples/flax/text-classification/run_flax_glue.py

```diff
@@ -468,8 +470,7 @@ def eval_step(state, batch):
         # train
         for batch in glue_train_data_collator(input_rng, train_dataset, train_batch_size):
             rng, dropout_rng = jax.random.split(rng)
```

Comment:
I think you also need to remove this line?

Reply:
Ah though maybe it won't make a difference because neither `rng` nor `dropout_rng` are used... Not totally sure. But regardless of whether this reduces speed, it's unnecessary and confusing to keep it around.